README.md

```diff
@@ -16,11 +17,17 @@ python time libraries mocking
 
 ### Usage
 
-`TimeTravel` is context manager patching all* time and I\O related modules in a single line.
+`TimeTravel` is context manager patching all* time and I\O related modules in a 
+single line. Inside the time travel context manager, the initial time is set to 
+86,400.0 seconds in order to support windows (this is the lowest acceptable 
```

Comment:
As you said, eventually the user doesn't need to see the specific time, so this can be in less significant place/

Reply:
I suggest we change that when it son't be significant... 😞

Reply:
At least put it in a different line.